torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -659,10 +661,14 @@ class TORCH_API ProcessGroupNCCL : public Backend {
   // Watchdog's inside loop.
   // Takes care of cleaning up completed work, and aborting upon failure or
   // timeout.
-  void workCleanupLoop();
+  void watchdogLoopHandler();
```

Comment:
nit: do the rename in a separate PR.  it'd be annoying to revert/reland the rename and its better to get ppl aligned and land it quickly